site/app/libraries/Core.php

```diff
@@ -162,6 +170,22 @@ public function setSessionManager(SessionManager $manager) {
         $this->session_manager = $manager;
     }
 
+    private function createEntityManager(AbstractDatabase $database): EntityManager {
```

Comment:
Does this need documentation above it for the parameter? (same with the rest of the following lines)

Reply:
My view that when we use inline type hints, we should only include a docstring if it includes additional type information that cannot be expressed inline. This would include stuff like union types (e.g. `string|int`) or concrete array elements (e.g. `string[]`) or if there's a template class (e.g. `Collection<Column>` here). Otherwise, there's no need for the docstring and it just adds a duplicate place we have to keep the types updated for no real benefit.

Reply:
Ok, seems reasonable enough then.